README.md

```diff
@@ -183,14 +183,17 @@ They live in [tests](tests). Run them with grunt: `grunt e2e`.
 
 ## Configuring Medic
 
-We ship with one "standard" configuration, which can be a useful basis to start with. It is located at [./config/standard](https://github.com/medic/medic/tree/master/config/standard).
+We ship with one "standard" configuration, which can be a useful basis to start with. It is located at [./config/standard](https://github.com/medic/medic/tree/master/config/standard). 
+
+This app is highly configurable and can be modified to suit your needs. Read the guide for developing community health applications if you would like to customize your application further.
 
 Configuration is performed using [Medic Configurer](https://github.com/medic/medic-conf). `medic-conf` expects a particular structure (seen in the standard config above). It compiles forms and configuration into the required formats, as well as uploading that configuration and performing other tasks.
 
 To import the standard configuration:
 
 1. Install medic-conf: `npm install -g medic-conf`
-2. Navigate to the configuration you want to import: `cd config/standard`
+2. Navigate to the configuration you want to import: `cd <medic-repo>/config/standard`
+1. Ensure the app/api is running. Specifically on localhost for these instructions. 
 3. Import the config: `medic-conf --url http://username:password@localhost:5984`
```

Comment:
This should reference the api port, so the docs should probably say `5988` not `5984`.

Reply:
As of medic-conf 2.0 this is `--url=http://username:password@localhost:5988`

Reply:
I've updated to use 5988. Looking at the code we replace 5984 with 5988.

Reply:
I think you still need the `=` sign though. `--url=http://...`